src/System.Text.RegularExpressions/tests/RegexGroupNameTests.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Xunit;
+using System;
+using System.Text.RegularExpressions;
+using System.Collections;
+
+namespace System.Text.RegularExpressionsTests
+{
+    /// <summary>
+    /// Tests the Name property on the Group class.
+    /// </summary>
+    public class RegexGroupNameTests
+    {
+        [Fact]
+        public static void NameTests()
```

Comment:
This test already tests that. There are 2 numbered groups here, this will output:
// Named Groups: 
//          0: 'The cow jumped over the moon.' 
//          1: 'the ' 
//          2: 'the' 
//          FirstWord: 'The' 
//          LastWord: 'moon' 
//          Punctuation: '.'